File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.py
def solution(babbling):
    x=['11','22','33','44']
    result=0
    for i in range(len(babbling)):
        babbling[i]=babbling[i].replace('aya','1').replace('ye','2').replace('woo','3').replace('ma','4')
        if babbling[i].isdigit():
            for j in range(4):
                flag=False
                if x[j] in babbling[i]:
                    flag=True
                    break
            if flag==False:
                result+=1
    return result

File: 프로그래머스/1/133499. 옹알이 （2）/옹알이 （2）.js
function solution(babbling) {
    let word=["aya","ye","woo","ma"];
    let result=0
    for(let i=0;i<babbling.length;i++){
        for(let j=0;j<4;j++){
            if(!babbling[i].includes(word[j].repeat(2)))
                babbling[i]=babbling[i].replaceAll(word[j]," ")
        }
        if(babbling[i].trim().length==0)
            result+=1
    }
    return result;
}